quantum/process_keycode/process_autocorrect.h

```diff
@@ -0,0 +1,17 @@
+// Copyright 2021 Google LLC
```

Comment:
Is the Google thing because it matches their OSS policy?

Reply:
It's because that's what the original code is.
https://github.com/getreuer/qmk-keymap/blob/a1f62991a99b205b7d493dcc1fdea3946cbb575c/features/autocorrection.c#L1-L13


And yeah, because of their OSS policies, IIRC>